src/OWML.Launcher/GameVersionHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using OWML.Common;
+
+namespace OWML.Launcher
+{
+    public class GameVersionHandler : IGameVersionHandler
+    {
+        private readonly IGameVersionReader _versionReader;
+        private readonly IModConsole _writer;
+        private readonly IModManifest _owmlManifest;
+
+        public GameVersionHandler(IGameVersionReader versionReader, IModConsole writer, IModManifest owmlManifest)
+        {
+            _versionReader = versionReader;
+            _writer = writer;
+            _owmlManifest = owmlManifest;
+        }
+
+        public void CompareVersions()
+        {
+            var gameVersionString = _versionReader.GetGameVersion();
+
+            _writer.WriteLine($"Game version: {gameVersionString}", MessageType.Info);
+            var isValidFormat = Version.TryParse(gameVersionString, out var gameVersion);
+            var minVersion = new Version(_owmlManifest.MinGameVersion);
```

Comment:
have you tested these for steam/epic versions? they are different

Reply:
nope, haven't. i don't own the steam version :D